corehq/apps/repeaters/models.py

```diff
@@ -419,6 +419,7 @@ class RepeatRecordAttempt(DocumentSchema):
     cancelled = BooleanProperty(default=False)
     datetime = DateTimeProperty()
     failure_reason = StringProperty()
+    success_response = StringProperty()
```

Comment:
would it make more sense to just have a generic `response` property instead? I guess then you'd have to write a migration...